From 9758dd744102dad11203e426c1509620d1e44e1f Mon Sep 17 00:00:00 2001
From: "ji.guo" <ji.guo@shmastermatrix.com>
Date: Mon, 16 Aug 2021 16:06:45 +0800
Subject: [PATCH] 444

---
 test1 | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test1 b/test1
index 3553e01..4dda16f 100644
--- a/test1
+++ b/test1
@@ -6,6 +6,7 @@ I20210810 22:30:20.354681 26009 :0] [Info][LiDAR:32]:[LiDAR] Constructor
 I20210810 22:30:20.354684 26009 :0] [Info][getBufferQueue:1445]:[LivoxManager] invalid index
 I20210810 22:30:20.354687 26009 :0] [Info][instantiateLidarLivox:73]:[LidarManager] new LidarLivox 1
 I20210810 22:30:20.354691 26009 :0] [Info][LiDAR:32]:[LiDAR] Constructor
+444
 I20210810 22:30:20.354693 26009 :0] [Info][getBufferQueue:1445]:[LivoxManager] invalid index
 I20210810 22:30:20.354696 26009 :0] [Info][instantiateLidarLivox:73]:[LidarManager] new LidarLivox 2
 I20210810 22:30:20.354701 26009 :0] [Info][LiDAR:32]:[LiDAR] Constructor
-- 
2.7.4

